### DISCUSSION TITLE
Documentation repo?

### DISCUSSION THREAD

**Discussion by tetious** [ArcadeData/arcadedb#95] [General]
I've found a few typos in the otherwise *excellent* documentation, and would like to contribute fixes. However, the "edit this section" links 404. 

Is the repo private? Is this by design? 😄

**Comment by lvca** [comment]
Hi @tetious, it would be awesome. I guess we forgot to change the visibility of the project (https://github.com/ArcadeData/arcadedb-docs) when we released ArcadeDB. Now it should be public.
